circle.yml

```diff
@@ -18,7 +18,7 @@ test:
     - pip install coala-bears colorama==0.3.6 --pre -U:
```

Comment:
please add a period after body, and its all set! :D

also, uses Closes instead of Fixes as this was not a bug :)